src/NServiceBus.Gateway.Tests/APIApprovals.Approve.approved.txt

```diff
@@ -13,7 +13,7 @@ namespace NServiceBus
     public class GatewaySettings
     {
         public void ChannelFactories(System.Func<string, NServiceBus.Gateway.IChannelSender> senderFactory, System.Func<string, NServiceBus.Gateway.IChannelReceiver> receiverFactory) { }
-        public void CustomRetryPolicy(System.Func<NServiceBus.Transports.IncomingMessage, System.Exception, int, System.TimeSpan> customRetryPolicy) { }
+        public void CustomRetryPolicy(System.Func<NServiceBus.Transport.IncomingMessage, System.Exception, int, System.TimeSpan> customRetryPolicy) { }
```

Comment:
To be consistent, we would also need to rename the `Retries` method to `DelayedRetries`. I think these names would be more intention revealing given recent changes in the core recoverability APIs.

@danielmarbach Do you agree to rename both methods?